src/main/java/meteordevelopment/meteorclient/systems/modules/render/BlockSelection.java

```diff
@@ -17,8 +18,11 @@
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.util.math.Box;
 import net.minecraft.util.math.Direction;
+import net.minecraft.util.math.Vec3d;
 import net.minecraft.util.shape.VoxelShape;
 
+import java.math.RoundingMode;
```

Comment:
Why are you adding imports that are never used? There is another one at the top.

Reply:
Forgot to remove the stuff

Reply:
I'm new to Mod Development and i needed to figure out what each field does and i just started messing around with different fields i got and in the end i found my current solution but now forgot to remove the old things